src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,16 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables) {
+        final StringBuilder builder = new StringBuilder();
+        for (Printable printable : printables) {
+            builder.append(printable.getPrintableString());
+            builder.append(" "); // Whitespace btw each printable
```

Comment:
isn't it apparent when you append an empty space? you can remove such redundant comments.

Reply:
Ok noted!